main.py

```diff
@@ -165,16 +194,34 @@ def beginRO(self, transactionName: str):
         T = Transaction(params[0], True, False, TM.time)
         self.transactions[transactionName] = T
 
+
+    # return True/False depending on whether the read was blocked
+    # print out the value of the variable if not blocked
     def read(self, transactionName: str, x: str):
         assert(transactionName in self.transactions)
         T = self.transactions[transactionName].RO
         if T.RO:
-            self.__readRO(T, x)
```

Comment:
Removing the __readRO and __read() functions for now, since they don't seem to help much